frontend/src/components/Divisions.tsx

```diff
@@ -94,15 +97,14 @@ const Divisions = ({
   onTabChange,
   onRefetch,
 }: DivisionsProps) => {
+  const [isDeleteModalOpen, setIsDeleteModalOpen] = useState<boolean>(false);
```

Comment:
This should not be requied as we need to send the row.id to the confirmation modal. I have explained it in the component.